workers/data_refinery_workers/processors/no_op.py

```diff
@@ -160,6 +161,14 @@ def _convert_genes(job_context: Dict) -> Dict:
 def _convert_affy_genes(job_context: Dict) -> Dict:
     """ Convert to Ensembl genes if we can"""
 
+    if 'internal_accession' not in job_context:
+        error_msg = "Told to convert AFFY genes without an internal_accession - how did this happen?"
```

Comment:
I'm not a fan of this "how did this happen" because it implies this is unexpected or that we don't know why it's happening, neither of which is the case.

Reply:
This is unexpected, since it should have been taken care of in the prepare files function.

Reply:
Ah okay, I thought that this was just about us not recognizing a platform, which is an expected error. However this is about `internal_accession` not being on `job_context`, which is weird and I cannot figure out how in the world it would ever happen.

Reply:
So then the only suggestion I have is that this should probably be checking for `if 'internal_accession' not in job_context or not job_context['internal_accession']:

Reply:
Addressed